docs/validators/key-management/multikey-nodes.md

```diff
@@ -163,10 +163,151 @@ At the first sight, this can be seen as a security degradation in terms of means
 
 Regarding point 3, each managed BLS key will create a virtual p2p identity that no node from the network can connect to since it does not advertise the connection info but is only used to sign p2p messages.
 Associated with a separate named identity, the system will make that BLS key virtually unreachable, and its origin hidden from the multikey nodes. Therefore, the node operators will need to apply the following changes on the `prefs.toml` file:
-* in the `[Preference]` section, the 2 options called `NodeDisplayName` and `Identity` will be changed to something relevant for the nodes that are running the multikey group;
+* in the `[Preference]` section, the 2 options called `NodeDisplayName` and `Identity` will be changed to something different used in the BLS' definitions to prevent easy matching. Generic names like `gateway` or `observer` are suitable for this section. 
```

Comment:
Generic names like `gateway` or `observer` are **NOT** suitable for this section?

Reply:
are suitable. As they are too generic to show you some links with the keys managed.

Reply:
added also the possibility of having random strings. Used random string in the example